src/lib/scheduler.py

```diff
@@ -1,6 +1,66 @@
-from datetime import datetime
-from src.lib.task import CarbonAwareTask
+from datetime import datetime, timedelta, timezone
+from lib.task import CarbonAwareTask
+import openapi_client
+from openapi_client.rest import ApiException
+
 
 def determine_optimal_execution_time(task: CarbonAwareTask) -> datetime:
     """Determines the optimal time to execute a CarbonAwareTask using the carbon aware sdk."""
-    pass
\ No newline at end of file
+
+    configuration = openapi_client.Configuration(
+    host = "http://localhost:5073",
```

Comment:
Maybe this will be deployed somewhere else once @seljaks has been able to deploy it :)

Reply:
So maybe make it modular with a constant at the top